src/overrides/safari.py

```diff
@@ -1,12 +1,17 @@
 from talon import Context, actions
+from ..overrides import safari_version  # only relative imports work
 
 ctx = Context()
 ctx.matches = r"""
 tag: browser
 app: safari
 """
 
+_SAFARI_VERSION = safari_version.get()
+_HOTKEY = "ctrl-shift-keypad_3" if _SAFARI_VERSION.startswith("18.") else "ctrl-shift-3"
+
+
```

Comment:
I don't understand the screaming snake case. These are not constants, they are calculated at runtime. If you rename the function like I suggest above you can simply do:

```python
_hotkey = "ctrl-shift-keypad_3" if get_safari_version().startswith("18.") else "ctrl-shift-3"